apis.md

```diff
@@ -9,3 +9,10 @@ The following OpenTracing APIs are currently available in the RFC stage:
 
 Please refer to the README files in the respective projects for examples of usage.
 
+### Implementation Notes
+
+OpenTracing APIs intend to express a similar experience to the user, while still being idiomatic to the language or framework. The following concepts apply to all variants.
+
+A **Span** is shared mutable state. It accumulates tags and logs, and can also create children. Finishing a span means capturing its state as a SpanRecord.
+
+A **SpanRecord** is an immutable representation of a completed span. It can be reported directly to an OpenTracing system, or encoded into a representation of an alternate system, such as HTrace or Zipkin.
```

Comment:
> I'm not sure, but seems if it were called Mock or FakeTracer, it would still allow folks to playground, and write test harnesses, but without the warranties.

@adriancole  Can you elaborate which features you would want to see in such Mock tracer?  And if we have example/zipkin-like converted to a real reference implementation, wouldn't it be sufficient as a playground? I am a bit uneasy about maintaining a full-featured "fake" implementation.

> Other tracers for zipkin exist in the OpenZipkin org, and have coupling there. I think it would make more sense for a new python zipkin tracer to exist as "zipkin-python" or similar there.. @abesto thoughts?

That's fine with me: `openzipkin/opentracing-zipkin-python`